18F/identity-idp
# LG-10233: Sign in "Use face or touch unlock" page revisions

## 🎫 Ticket

[LG-10233](https://cm-jira.usa.gov/browse/LG-10233)

## 🛠 Summary of changes

Updates content and experience for Face or Touch Unlock (and Security Key) authentication during sign-in:

- Green lock screen goes away, redirects immediately
- "Remember browser" button shown before authentication
- New content below spinner ("Authenticating your credentials…")
- Troubleshooting module on use f/t unlock sign in flow

## 📜 Testing Plan

1. Sign in with an account that has Security Key or Face or Touch Unlock configured
2. Observe updated verification page in sign-in

## 👀 Screenshots

Authenticator|Before|After
---|---|---
Security Key|![Screen Shot 2023-07-10 at 2 17 59 PM](https://github.com/18F/identity-idp/assets/1779930/01a8403e-a3a8-4994-80b4-4bdacf20a557)|![Screen Shot 2023-07-10 at 2 17 43 PM](https://github.com/18F/identity-idp/assets/1779930/b9f9096b-a610-476b-879c-7f2c263b7de4)
Face or Touch Unlock|![Screen Shot 2023-07-10 at 2 18 36 PM](https://github.com/18F/identity-idp/assets/1779930/51d09eda-abcb-4291-8dab-5c9aab417e61)|![Screen Shot 2023-07-10 at 2 18 29 PM](https://github.com/18F/identity-idp/assets/1779930/3d9e9fc1-3e58-4a6c-9ffe-14a7cd4e5b3b)
